src/sagas/SleepingPillApi.js

```diff
@@ -11,3 +13,17 @@ export const sessionBySlug: (string) => any =
   (slug) => fetch(`${config.urls.sleepingPill}/public/allSessions/${slug}`)
     .then((res) => res.json())
 
+const options = {
+	"Content-Type": "application/json",
+    headers: {
+      'Content-Type': "application/json",
+      "Voter-ID": DeviceInfo? "javazone2017" : "javazone2017"
```

Comment:
The `Voter-ID` need to be unique for each device. If `react-native` doesn't have a way to extract one then we need to just generate one and store it (about the same way we're storing settings today)